lms/djangoapps/courseware/views/views.py

```diff
@@ -25,6 +25,7 @@
     QueryDict,
 )
 from django.shortcuts import redirect
+from django.template.loader import render_to_string
```

Comment:
Looks like this wouldn't even do anything because we were importing `render_to_string` from `edxmako` down below on L78. Removed